distribution/docs/pom.xml

```diff
@@ -24,6 +24,37 @@
     <properties>
         <skipTests>true</skipTests>
     </properties>
+        <dependencies>
+        <dependency>
```

Comment:
❓ Is this dependency required? I'd think you'd only need the classified one below.